packages/block-editor/src/components/rich-text/index.js

```diff
@@ -214,13 +214,19 @@ export function RichTextWrapper(
 
 			const { getBlockAttributes } = select( blockEditorStore );
 			const blockAttributes = getBlockAttributes( clientId );
-			const fieldsList = blockBindingsSource?.getFieldsList?.( {
-				select,
-				context: blockBindingsContext,
-			} );
+			let clientSideFieldLabel = null;
+			if ( blockBindingsSource?.editorUI ) {
+				const editorUIResult = blockBindingsSource.editorUI( {
+					select,
+					context: blockBindingsContext,
+				} );
+				clientSideFieldLabel = editorUIResult.data?.find(
+					( item ) => item.key === relatedBinding?.args?.key
```

Comment:
needs `isSelected` call
```suggestion
					( item ) =>
						source.isSelected( { item, binding } ) ??
						item.key === args?.key
```